misc_scripts/dangerous/manual_secondary_cleanup.py

```diff
@@ -18,7 +18,7 @@
 import glob
 
 # Meerpipe imports
-sys.path.append('/fred/oz005/users/acameron/pipeline_stuff/andrew_meerpipe_dev/meerpipe/')
```

Comment:
This path amendment is required to access the db_utils library. Happy to discuss ideas for a more effective way of implementing this, I agree it's not ideal.